### PR TITLE
Create card type

### PR DESCRIPTION
**Pull request by Caleb-Nieto**
### Feature / Bug Description
Create card type

### Solution
RegisterService.ts and RegiterForm.tsx  added, ModalForm implemented

### Notes
For now, the responsible select is static in the front end

### Tickets
[WMA-37](https://cdentalcaregroup.atlassian.net/browse/WMA-37)

### Screenshots / Screencasts
Captura de pantalla 2024-06-06 151728
Captura de pantalla 2024-06-06 150837

[WMA-37]: https://cdentalcaregroup.atlassian.net/browse/WMA-37?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/pages/cardtypes/components/RegisterCardTypeForm.tsx [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+interface FormProps {
+  form: FormInstance;
+}
+
+const RegisterCardTypeForm = ({ form }: FormProps) => {
+  return (
+    <Form form={form}>
+      <div className="flex flex-col">
+        <div className="flex flex-row justify-between flex-wrap">
```
Comment (imadiaz) [inline src/pages/cardtypes/components/RegisterCardTypeForm.tsx:18]: can u use strings here?

File: src/pages/cardtypes/components/RegisterCardTypeForm.tsx [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+          >
+            <Input
+              size="large"
+              addonBefore={<FiTool />}
+              maxLength={25}
+              placeholder={Strings.methodology}
+            />
+          </Form.Item>
+          <Form.Item
+            name="name"
```
Comment (imadiaz) [inline src/pages/cardtypes/components/RegisterCardTypeForm.tsx:35]: same here

**Review by imadiaz — APPROVED** [review]
lgtm just some comments

File: src/pages/cardtypes/components/RegisterCardTypeForm.tsx (hunk 2 above)
Comment (Caleb-Nieto) [reply src/pages/cardtypes/components/RegisterCardTypeForm.tsx:35]: I think no, because those are the form values, all the forms are like that

**Pull request merged**